packages/headless-react/tsconfig.test.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "allowUnreachableCode": false,
+    "allowSyntheticDefaultImports": true,
+    "allowJs": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "esModuleInterop": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "removeComments": false,
+    "sourceMap": true,
+    "declaration": true,
+    "jsx": "react-jsx",
+    "outDir": "dist/",
+    "moduleResolution": "Node",
+    "module": "ES2022",
+    "target": "ES2022",
+    "lib": ["dom", "ES2022"],
+    "types": ["jest", "react", "react-dom"]
```

Comment:
Needs to specify types to not carry around jasmine types. They break React Testing Library assertions.